packages/app/src/specs/SpecsList.vue

```diff
@@ -64,6 +68,7 @@
                 :depth="row.data.depth - 2"
                 :style="{ paddingLeft: `${((row.data.depth - 2) * 10) + 16}px` }"
                 :indexes="getDirIndexes(row.data)"
+                :aria-controls="getIdIfDirectory(row)"
```

Comment:
Just a general Vue question, is there a way to execute arbitrary code from within a template loop? Say, if `getIdIfDirectory()` were a lot more expensive than it currently is and you only want to execute it once. Or would you just `compute` your data ahead of time and iterate over that instead?

Reply:
You're right, you'd have a `computed` that uses `map` or whatever to make a new set of rows with the extra attribute, and that would be cached until, say, the data updates when a file is added.